src/content/guides/code-splitting.md

```diff
@@ -253,7 +255,7 @@ Entrypoint index = index.bundle.js
 ...
 ```
 
-As `import()` returns a promise, it can be used with [`async` functions](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/async_function). However, this requires using a pre-processor like Babel and the [Syntax Dynamic Import Babel Plugin](https://babeljs.io/docs/plugins/syntax-dynamic-import/#installation). Here's how it would simplify the code:
+As `import()` returns a promise, it can be used with [`async` functions](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/async_function). However, this requires using a pre-processor like Babel and the [Syntax Dynamic Import Babel Plugin](https://babeljs.io/docs/plugins/syntax-dynamic-import/#installation). For `async await` `babel-polyfill` is required. Common usage can be found [here] (https://babeljs.io/docs/en/babel-polyfill/). Or in case you are using mutiple entries.
```

Comment:
I think this isnt ready, can you see this suggestion and maybe apply or improve from here:
```suggestion
As `import()` returns a promise, it can be used with [`async` functions](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/async_function). However, this requires using a pre-processor like Babel and the [Syntax Dynamic Import Babel Plugin](https://babeljs.io/docs/plugins/syntax-dynamic-import/#installation). In order to use `async` and `await`, `babel-polyfill` is required. Common usage examples can be found [here] (https://babeljs.io/docs/en/babel-polyfill/).
```

Also i have removed the last sentence: "Or in case you are using mutiple entries.", it should make more sense, can you explain exactly what you mean here? Are you saying you need `babel-polyfill` if user had configured multiple entry points?